content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -65,8 +65,7 @@ _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traff
 snip_adding_your_application_to_the_ambient_mesh_1
 
 # test traffic after ambient mode is enabled
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 "$snip_adding_your_application_to_the_ambient_mesh_2_out"
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
+_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 "namespace/default labeled"
```

Comment:
Why aren't you testing snips 1, 3, 5, and 6?

Also, since you're just testing the doc, not the function, you don't need to `verify_` on commands that the doc doesn't show what to expect:
```suggestion
snip_adding_your_application_to_the_ambient_mesh_2
```

Reply:
@frankbu I generated the snips using `make gen` and just adjusted for the content but the content I want to add is the actual ztunnel log verification to educate the user on the new in-pod redirection so the output to be matched is not always the same being the reason why I skipping some and some of those where skipped from the previous version of the doc. Let me know if I'm missing anything.

Reply:
I agree. Understood.